build/images/scripts/start_ovs_windows_containerd.ps1

```diff
@@ -0,0 +1,167 @@
+$ErrorActionPreference = "Stop"
```

Comment:
Do you plan to support "containerd-ovs" CI pipeline in this PR? I remember it is mentioned in issue https://github.com/antrea-io/antrea/issues/5249

Reply:
I have created new 'pristine-win-initial' snapshot. In this environment, I removed ovs driver and userspace program.

Reply:
Thanks @XinShuYang , I think we can deploy a new testbed without ovs driver and userspace program pre-installed and add it to ci pipeline , and add new jobs w.r.t to 'containerd-ovs' ... I am working on both parts.

Reply:
I am okay with creating new jobs for this scenario if you want, but they can share the same testbed which has multiple snapshot for different test environment.

Reply:
Using snapshots can help conserve resources by allowing to reuse the same testbed, @NamanAg30 , however before using snapshot meant for ovs container case, we should assure that ovs driver and ovs userspace processes running on host from previous snapshot/installations is cleanup and checked upon.

Reply:
@NamanAg30 

> Do you plan to support "containerd-ovs" CI pipeline in this PR? I remember it is mentioned in issue #5249

Any update?

Reply:
@NamanAg30 , 
 To work on a separate PR wrt CI changes #5325